bust-a-move-framework/src/main/java/bustamove/gamestate/Button.java

```diff
@@ -54,11 +50,6 @@
      */
     public Button(final String text, final float y, final float width,
                   final float height) {
-        assert text != null;
```

Comment:
Asserts aren't enabled during runtime by default. According to [this](http://stackoverflow.com/questions/4537609/is-it-a-good-practice-to-use-assert-in-java) they should only be used during debugging. I don't think we are going to be adding changes to the Button class, so they won't be needed anymore.